### ISSUE TITLE
Terser plugin uses deprecated 'transformBundle' Rollup hook

### ISSUE THREAD

**Issue by Kinrany** [sveltejs/sapper-template#107]
```
$ npm run build

> The transformBundle hook used by plugin terser is deprecated. The renderChunk 
hook should be used instead.
```

~~This is almost certainly a problem with [rollup-plugin-terser](https://github.com/TrySound/rollup-plugin-terser).~~

Wait, no, it's just that the dependency is out of date: [package.json](https://github.com/sveltejs/sapper-template/blob/rollup/package.json) references v1.0.1 while the latest version is v4.0.2

**Comment by Seb35** [comment]
This was fixed by #113. Is there a policy to backport it into master?

**Comment by Kinrany** [comment]
Fixed by https://github.com/sveltejs/sapper-template/issues/113
